lib/cli/scripts/resources.ts

```diff
@@ -19,7 +19,7 @@
 export const ACTIVITI_CLOUD_APPS: any = {
     SUB_PROCESS_APP: {
         name: 'subprocessapp',
-        file_location: 'https://github.com/Alfresco/alfresco-ng2-components/blob/develop/e2e/resources/activiti7/subprocessapp.zip?raw=true',
+        file_location: (TAG = 'develop') => `https://github.com/Alfresco/alfresco-ng2-components/blob/${TAG}/e2e/resources/activiti7/subprocessapp.zip?raw=true`,
```

Comment:
Previously we were using it as a normal property, now as a function. How it doesn't cause any problem with other parts of the app?

Reply:
this file is only used by init-aae-env command

Reply:
in this case: 🎖️

Reply:
what a pleasure!